src/main/kotlin/com/github/zzehring/intellijjsonnet/JsonnetLSPreloadingActivity.kt

```diff
@@ -181,8 +183,14 @@ object JsonnetLSPreloadingActivity : PreloadingActivity() {
     }
 
     private fun setExecutablePerms(file: File) {
-        val executablePerms = PosixFilePermissions.fromString("rwxr--r--")
-        Path(file.toString()).setPosixFilePermissions(executablePerms)
+        if (Files.getFileAttributeView(file.toPath(), PosixFileAttributeView::class.java) != null) {
```

Comment:
This check is taken from JDK code, difference is that we will fallback instead of throwing an exception.